styleguide/source/_patterns/02-organisms/00-global/06-wayfinder/wayfinder.js

```diff
@@ -11,12 +11,20 @@
   Drupal.behaviors.wayfinder = {
     attach: function (context, settings) {
       (function ($) {
+
+        var pageTitle  = document.title.replace(' | AMA', '');
```

Comment:
@designerneil this looks like it's the debugging code you mention in the testing instructions? Also the console.log line shouldn't be committed.

Reply:
good catch. It's been removed. Thanks